views/private_subnet/show.erb

```diff
@@ -57,13 +57,15 @@
           <tr>
             <td class="whitespace-nowrap py-4 pl-4 pr-3 text-sm font-medium text-gray-900 sm:pl-6" scope="row"><%= nic[:name] %></td>
             <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
-              <span class="copy-content cursor-pointer" data-content="<%= nic[:private_ipv4] %>" data-message="Copied Private IPv4">
+              <span class="copy-content" data-content="<%= nic[:private_ipv4] %>" data-message="Copied Private IPv4">
                 <%= nic[:private_ipv4] %>
+                <img src="/icons/hero-clipboard-document.svg" width="20px" class="inline-block copy-button cursor-pointer"/>
```

Comment:
```suggestion
              <span class="copy-content flex items-center" data-content="<%= nic[:private_ipv4] %>" data-message="Copied Private IPv4">
                <%= nic[:private_ipv4] %>
                <img src="/icons/hero-clipboard-document.svg" width="18px" class="inline-block copy-button cursor-pointer"/>
```
The icon slightly overflows the text container. It could be vertically centered.

**Before:**
<img width="547" alt="Screenshot 2023-11-10 at 23 12 44" src="https://github.com/ubicloud/ubicloud/assets/993199/18ede6fa-abc9-476e-a10c-c6538f330400">

**After:**
<img width="504" alt="Screenshot 2023-11-10 at 23 14 04" src="https://github.com/ubicloud/ubicloud/assets/993199/c99db923-fd30-4bde-9cdb-13d113369663">